docs/sources/release-notes/v2.3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Grafana Mimir version 2.3 release notes"
+menuTitle: "V2.3 release notes"
+description: "Release notes for Grafana Mimir version 2.3"
+weight: 300
+---
+
+# Grafana Mimir version 2.3 release notes
+
+Grafana Labs is excited to announce version 2.3 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.2, there is upgrade-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for OpenTelemetry Metrics ingestion path**:
+  Grafana Mimir now has experimental support for ingesting metrics from the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) via two options: `prometheusremotewrite` and `otlphttp`. There is also experimental support for the OpenTelemetry Metrics ingestion path on `/otlp/v1/metrics`.
+
+- **Added support to split instant queries by time**:
+  As an experimental feature, instant queries in Grafana Mimir now will get the same benefits of query sharding we see for range queries. When enabled, splitting instant queries by time will contribute in some cases to an exponential speed up in instant query response time.
+
+- **Object storage can now be configured for all components**:
+  Object storage can now be configured for all Grafana Mimir components using the `common` YAML config option key (or `-common.storage.*` CLI flags), this enhancement reduces the total configuration overhead for running Mimir with object storage and centralizes your object storage configuration in one place reducing errors.
+
+- **Add packaging for both debian based deb file and redhat based rpm**:
+  With Grafana Mimir 2.3 we will be publishing a deb file and a rpm file of Grafana Mimir to make installation and running on debian or redhat based systems simpler. This was made possible by a community contribution from wilfriedroset.
```

Comment:
When the release note will be read, the release assets will be already published.

```suggestion
  With Grafana Mimir 2.3 we're publishing a deb file and a rpm file of Grafana Mimir to make installation and running on debian or redhat based systems simpler. This was made possible by a community contribution from wilfriedroset.
```